### PR TITLE
product page with filter

### PR DESCRIPTION
**Pull request by rahul0396**
https://user-images.githubusercontent.com/43301962/154839324-741575ed-4064-4644-967b-3db22c3ae22d.mp4

### CODE REVIEW

**Review by Ronak-Mutha — COMMENTED** [review]
Remove extra comments and unnecessary spaces and you can use variables for colors and spacing. UI looks clean and good to merge.
Note: These are just suggestions, try to work on them after submission [Not required, good to have].

File: pages/product.html [hunk 1]
```diff
@@ -0,0 +1,298 @@
...
+                <strong>₹299</strong>
+                </span>
+                <span class="earlier-price">₹599</span>
+                <span class="pct-off">50% off</span>
+            </div>
+            <button class="btn btn-primary-solid">Add to cart</button>
+        </div>
+        <div class="card card-vertical product-card">
+            <!-- <div class="card-vertical-header">
+                <span class="status-badge">In stock </span> -->
```
Comment (saadahmadhere) [inline pages/product.html:173]: unwanted comments

File: pages/product.html [hunk 2]
```diff
@@ -0,0 +1,298 @@
...
+        <div class="card card-vertical product-card">
+            
+            <button class="card-icon-btn ml-auto">
+                <i class="far fa-heart m-auto"></i>
+                </button>
+            <img class="card-vertical-img" src="/assets/image/Deep Sea Fish oil 1.jpg" alt="..." />
+    
+            <h1 class="card-head">Healthy by nature Peanut Butter</h1>
+            
+
```
Comment (saadahmadhere) [inline pages/product.html:208]: unwanted space.

**Review by saadahmadhere — APPROVED** [review]
Nice and clean code. Would be good if the space and unwanted comments are removed. For now, it's good to merge for submissions.

**Pull request merged**